src/features/vote/hooks/usePriceRoundEvents.ts

```diff
@@ -0,0 +1,34 @@
+import provider from "common/utils/web3/createProvider";
+import createVotingContractInstance from "web3/createVotingContractInstance";
+import { useQuery } from "react-query";
+
+import {
+  queryPriceRoundEvents,
+  PriceRound,
+} from "web3/queryVotingContractEvents";
+
+const signer = provider.getSigner();
+const contract = createVotingContractInstance(signer);
+
+// This can be accessed without logging the user in.
+export default function usePriceRoundEvents() {
+  const { data, error, isFetching } = useQuery<PriceRound[]>(
+    "priceRoundEvents",
+    () => {
+      return queryPriceRoundEvents(contract).then((res) => {
+        if (res instanceof Error) return [];
```

Comment:
dont think res could be an error unless you are just returning an error rather than throwing, which would not be good.  I would suggest in these useQuery style calls, we just let the function throw errors and form a strategy to handle it holistically since useQuery provides an error state.  this would simplify your `then` handler, as you wont need to check for a null condition. we might have to come up with a convention for this, wdyt?